packages/app/src/cli/commands/app/config/push.ts

```diff
@@ -10,9 +13,17 @@ export default class ConfigPush extends Command {
   static flags = {
     ...globalFlags,
     ...appFlags,
+    config: Flags.string({
+      hidden: false,
+      description: 'Name for the config file.',
+      env: 'SHOPIFY_FLAG_APP_CONFIG',
+    }),
   }
 
   public async run(): Promise<void> {
     const {flags} = await this.parse(ConfigPush)
+    const app = await loadApp({specifications: [], configName: flags.config, directory: flags.path, mode: 'report'})
```

Comment:
Maybe: I _think_ this should be `mode: strict`, as we wouldn't want to push a broken app. Report I think tries to fill the blanks if the file is messed up.